docker-dev.yaml

```diff
@@ -43,23 +43,23 @@ services:
 
   rbac-server:
     build:
-      dockerfile: rbac/server/Dockerfile-dev
+      dockerfile: docker/dev/rbac.server.Dockerfile
     container_name: rbac-server-dev
     image: rbac-server-dev:${ISOLATION_ID-latest}
     volumes:
       - ".:/project/tmobile-rbac"
 
   rbac-tp:
     build:
-      dockerfile: rbac/processor/Dockerfile-dev
+      dockerfile: docker/dev/rbac.processor.Dockerfile
     container_name: rbac-tp-dev
     image: rbac-tp-dev:${ISOLATION_ID-latest}
     volumes:
       - ".:/project/tmobile-rbac"
 
   ledger-sync:
     build:
-      dockerfile: rbac/ledger_sync/Dockerfile-dev
+      dockerfile: docker/dev/rbac.ledger_sync.Dockerfile
```

Comment:
It appears that you are replacing a different dockerfile here.  Dockerfile-Dev is different the Dockerfile.  Can you confirm that they are the same?